src/NuGet.Services.AzureSearch/Db2AzureSearch/NewPackageRegistrationProducer.cs

```diff
@@ -296,5 +297,21 @@ public PackageRegistrationRange(int minKey, int? maxKey, int packageCount)
             /// </summary>
             public int PackageCount { get; }
         }
+
+        public class PackageRegistrationInfo
```

Comment:
This isn't use by anyone else other than `NewPackageRegistrationProducer` right? Should this be `private`?

Reply:
Yeah, nice catch.